src/index.html

```diff
@@ -22,6 +22,446 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo-of-Moyo"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-li">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__image-container"></div>
+
+          <div class="card__description">
+            <h3 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__id">Product code: 195434</p>
+
+            <div class="card__rating">
+              <div class="stars">
+                <div class="stars__star stars__star--1"></div>
+
+                <div class="stars__star stars__star--2"></div>
+
+                <div class="stars__star stars__star--3"></div>
+
+                <div class="stars__star stars__star--4"></div>
+
+                <div class="stars__star stars__star--5"></div>
+              </div>
```

Comment:
```suggestion
              <div class="stars stars__star--4">
                <div class="stars__star"></div>

                <div class="stars__star"></div>

                <div class="stars__star"></div>

                <div class="stars__star"></div>

                <div class="stars__star"></div>
              </div>
```
that should be done via modifier